### DISCUSSION TITLE
Tidal-HiFi

### DISCUSSION THREAD

**Discussion by sykesgabri** [catppuccin/catppuccin#2516] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

Within the settings of Tidal-HiFi (I am referring to the Linux port available here https://github.com/Mastermindzh/tidal-hifi, which is the Tidal web client running in Electron), there is a page titled "Theming" that allows for either directly writing custom CSS, or uploading a pre-written CSS file.

### Any additional comments?

I have checked the Discussions and Issues (both open and closed) for mentions of Tidal and found nothing. There also don't appear to be any ports that have already been done but not added to the official list.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by pernydev** [comment]
Would love this!

**Comment by pernydev** [comment]
I made something...
![image](https://github.com/user-attachments/assets/f3c2e175-3213-4409-91e6-15ebe00b5847)

**Reply by pernydev** [reply]
Just made a repo for it! https://github.com/pernydev/tidal-catppuccin
